pkgs/by-name/da/date/package.nix

```diff
@@ -0,0 +1,26 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "date";
+  version = "3.0.1";
+
+  src = fetchFromGitHub {
+    owner = "HowardHinnant";
+    repo = "date";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-ZSjeJKAcT7mPym/4ViDvIR9nFMQEBCSUtPEuMO27Z+I=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  meta = {
```

Comment:
Missing `platforms`.

Reply:
This has now been resolved. I cannot resolve my own reviews on other peoples' PRs.